_includes/head.html

```diff
@@ -10,6 +10,10 @@
         <meta property="og:url" content="{{ site.baseurl }}" />
         <meta property="og:title" content="The Open Microscopy Environment" />
         <meta property="og:description" content="A consortium of universities, research labs, industry and developers producing open-source software and format standards for microscopy data." />
+        <meta name="twitter:title" content="The Open Microscopy Environment">
+        <meta name="twitter:description" content="A consortium of universities, research labs, industry and developers producing open-source software and format standards for microscopy data.">
+        <meta name="twitter:image" content="{{ site.baseurl }}">
+        <meta name="twitter:card" content="{{ site.baseurl }}/img/logos/ome-logo_fb.png">
```

Comment:
This image location needs to be under "image" and the content of the "card" has to be "summary"

Not sure if the order matters too, but https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/summary has the "card" item first

Reply:
Also need to full URL not the site.baseurl code

Reply:
Okay, I've pushed changes in the next commit.